### DISCUSSION TITLE
Irrelevant News

### DISCUSSION THREAD

**Discussion by ddale33** [tmrev/tmrev.io#85] [Ideas]
Indiana Jones and The Dial of Destiny has a news article for the game Destiny 2

**Comment by KegenGuyll** [comment]
I agree there is a lot of "irrelevant" news; the real issue is the lack of news articles. There needs to be some cron job created that will run the Kubrick news engine to find new articles. Currently, the process is manually done and that is why there are a lot of out-of-date articles or why you see a "Destiny 2" article on an Indiana Jones movie. I would like to hear your thoughts on this issue.  @ddale33
